src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+
+type Props = {
+  isHidden: boolean;
+  setIsHidden: (arg: boolean) => void;
```

Comment:
Correct, but it's better to type `setState` function returned from `useState` as `Dispatch<SetStateAction<boolean>>`, so you will be able to use `setIsHidden `like `setIsHidden((previousValue) => someReturn)`

Reply:
Didn't know about this possibility (Dispatch<SetStateAction<boolean>>). It will be useful. Thanks!